src/app/(auth)/signup/page.tsx

```diff
@@ -29,49 +31,83 @@ export default function SignUp() {
     if (error) {
       throw new Error(`An error occurred trying to sign up: ${error.message}`);
     }
-    push('/');
+    setEmailSentCount(1);
+    // push('/verify-email');
```

Comment:
Do you think we'll need these comments? If not, should we get rid of them?

Reply:
since it seems like we may implement the stretch (push to `/signup?sent=true` as you suggested) in the future, i'll leave the comments for now?